Gemfile

```diff
@@ -5,6 +5,6 @@ gemspec
 gem "rails", "~> #{ENV["RAILS_VERSION"]}" if ENV["RAILS_VERSION"]
 
 # test/unit is no longer bundled with Ruby 2.2, but required by Rails
-if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new("2.2.0")
+if Gem::Version.new(RUBY_VERSION.dup) >= Gem::Version.new("2.2.0")
```

Comment:
I was getting an error where `Gem::Version` would attempt to `strip!` the version string otherwise. Was at the same time as having trouble with my 1.9.3 build though so probably related to that.